### PR TITLE
Use secure password for psql

### PR DESCRIPTION
**Pull request by tim-lo**

### CODE REVIEW

File: DrivewayDentDeletion/Operators/configure-postgres.sh [hunk 1]
```diff
@@ -12,29 +12,60 @@ function usage {
...
   esac
 done
 
-namespace=$(echo $namespace | sed 's/-/_/g')
-postgresPod=$(oc get pod -n postgres -l name=postgresql -o jsonpath='{.items[].metadata.name}')
+DB_POD=$(oc get pod -n postgres -l name=postgresql -o jsonpath='{.items[].metadata.name}')
+DB_SVC="$(oc get cm postgres-config -ojson | jq '.data["postgres.env"] | split("\n  ")' | grep DATABASE_SERVICE_NAME | cut -d "=" -f 2- | tr -dc '[a-z0-9-]\n').postgres.svc.cluster.local"
+DB_USER=$(echo $NAMESPACE | sed 's/-/_/g')
+DB_NAME=db_${DB_USER}
+DB_PASS=$(LC_ALL=C tr -dc 'A-Za-z0-9' </dev/urandom | head -c 32 ; echo)
```
Comment (hrizvi1) [inline DrivewayDentDeletion/Operators/configure-postgres.sh:30]: LC_ALL=C tr -dc 'A-Za-z0-9' </dev/urandom | head -c 32 ; echo ?
Whats happening here

File: DrivewayDentDeletion/Operators/configure-postgres.sh [hunk 2]
```diff
@@ -12,29 +12,60 @@ function usage {
...
       ;;
     \? ) usage; exit
       ;;
   esac
 done
 
-namespace=$(echo $namespace | sed 's/-/_/g')
-postgresPod=$(oc get pod -n postgres -l name=postgresql -o jsonpath='{.items[].metadata.name}')
+DB_POD=$(oc get pod -n postgres -l name=postgresql -o jsonpath='{.items[].metadata.name}')
+DB_SVC="$(oc get cm postgres-config -ojson | jq '.data["postgres.env"] | split("\n  ")' | grep DATABASE_SERVICE_NAME | cut -d "=" -f 2- | tr -dc '[a-z0-9-]\n').postgres.svc.cluster.local"
```
Comment (hrizvi1) [inline DrivewayDentDeletion/Operators/configure-postgres.sh:27]: oc get cm postgres-config -ojson  <-- missing a space here

File: DrivewayDentDeletion/Operators/configure-postgres.sh (hunk 1 above)
Comment (tim-lo) [reply DrivewayDentDeletion/Operators/configure-postgres.sh:30]: random string generation for the password

File: DrivewayDentDeletion/Operators/configure-postgres.sh (hunk 2 above)
Comment (tim-lo) [reply DrivewayDentDeletion/Operators/configure-postgres.sh:27]: space between option and argument is optional, but i'll add a space for consistency

File: DrivewayDentDeletion/Operators/configure-postgres.sh (hunk 1 above)
Comment (hrizvi1) [reply DrivewayDentDeletion/Operators/configure-postgres.sh:30]: okay

File: DrivewayDentDeletion/Operators/configure-postgres.sh (hunk 2 above)
Comment (hrizvi1) [reply DrivewayDentDeletion/Operators/configure-postgres.sh:27]: yes please

**Pull request merged**